tools/bump_version.sh

```diff
@@ -37,9 +37,9 @@ update_dep_version() {
 update_image_versions() {
     echo "Updating file $1"
     # Update the version of the development container
-    sed -i "s/asterinas\/asterinas:[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+/asterinas\/asterinas:${new_version}/g" $1
+    sed -i "s/asterinas\/asterinas:[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+\(-[[:digit:]]\+\)\?/asterinas\/asterinas:${new_version}/g" $1
     # Update the test environment described in the OSDK manual
-    sed -i "s/asterinas\/osdk:[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+/asterinas\/osdk:${new_version}/g" $1
+    sed -i "s/asterinas\/osdk:[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+\(-[[:digit:]]\+\)\?/asterinas\/osdk:${new_version}/g" $1
```

Comment:
Isn't this a separate bugfix? It has nothing to do with Github tags. If so, this commit title should NOT mention "Github tag". This is misleading. Put this change into another commit and give a proper title.